Sources/ProjectDescription/TargetAction.swift

```diff
@@ -41,12 +57,22 @@ public struct TargetAction: Codable {
     ///   - path: Path to the script to execute.
     ///   - order: Target action order.
     ///   - arguments: Arguments that to be passed.
-    init(name: String, tool: String?, path: String?, order: Order, arguments: [String]) {
+    ///   - inputPaths: List of input file paths.
+    ///   - inputFileListPaths: List of input filelist paths.
+    ///   - outputPaths: List of output file paths.
+    ///   - outputFileListPaths: List of output filelist paths.
+    init(name: String, tool: String?, path: String?, order: Order, arguments: [String],
```

Comment:
Nitpick: I'd organize the arguments in different lines.

Reply:
Same comment for all the constructors where the new arguments have been added.

Reply:
I've corrected these and pushed up to the branch